data/SIMS Activation Log.csv

```diff
@@ -0,0 +1,34 @@
+Year,Country,Code,Response,Support,Significance (1-5; 5 as most significant activations),Start Date,End Date
```

Comment:
@pknight03 can you limit the scope to a google sheet that *only* has the data we want.